tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -48,11 +50,21 @@ def get_changed_connector_names() -> Set[str]:
         if file_path.startswith(SOURCE_CONNECTOR_PATH_PREFIX)
     }
 
-    def get_connector_name_from_path(path):
-        return path.split("/")[2]
-
     return {get_connector_name_from_path(changed_file) for changed_file in changed_source_connector_files}
 
+def get_changed_acceptance_test_config() -> Set[str]:
+    """Retrieve a list of connector names for which the acceptance_test_config file was changed in the current branch (compared to master).
+
+    Returns:
+        Set[str]: Set of connector names e.g {"source-pokeapi"}
+    """
+    changed_acceptance_test_config_paths = {
+        file_path
+        for file_path in AIRBYTE_REPO.git.diff("--name-only", "origin/master...").split("\n")
+        if file_path.startswith(SOURCE_CONNECTOR_PATH_PREFIX) and file_path.endswith(ACCEPTANCE_TEST_CONFIG_FILE_NAME)
```

Comment:
cool use of git to see if the file changed!

Reply:
The current limitation is that it checks if the `acceptance-test-config.yml` was modified, not specifically that `disable-for-version` was added. We'd be required a review even if the `disable-for-version` key was added in previous PRs and the change to `acceptance-test-config.yml` are not related to backward compatibility. I'll try to check the diff content itself.